app/src/test/kotlin/deliveryfee/AppTest.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
+ */
+package deliveryfee
+
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+import kotlin.test.assertEquals
+
+// https://github.com/woltapp/engineering-summer-intern-2023
+
+
+class testEachFunctions {
+
+//
+    @Test fun OrderSurcharge() {
```

Comment:
You can declare it like this to make it more understandable what you want to test. 
```kotlin

fun `calculateOrderSurcharge should be ... when ....`

```